examples/default-values-backend/src/setup.ts

```diff
@@ -74,83 +88,55 @@ const isOk = (status: number) => {
 };
 
 async function createAppDefinition() {
-  const body = {
-    name: 'Default field values',
-    src: `${HOSTED_APP_URL}/frontend`,
-    locations: [
+  try {
+    const appDefinitionProps = await client.appDefinition.create(
       {
-        location: 'app-config',
+        organizationId: ORG_ID,
       },
-    ],
-  };
-
-  const response = await nodeFetch(`${BASE_URL}/organizations/${ORG_ID}/app_definitions`, {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/vnd.contentful.management.v1+json',
-      Authorization: `Bearer ${CMA_TOKEN}`,
-    },
-    body: JSON.stringify(body),
-  });
-
-  if (isOk(response.status)) {
-    const j = await response.json();
-    console.log(`Created app definition. APP_ID is ${j.sys.id}`);
-    return j.sys.id;
-  } else {
-    throw new Error('App definition creation failed: ' + (await response.text()));
+      {
+        name: 'Default field values',
+        src: `${HOSTED_APP_URL}/frontend`,
+        locations: [{ location: 'app-config' }],
+      }
+    );
+    console.log(`Created app definition. APP_ID is ${appDefinitionProps.sys.id}`);
+    return appDefinitionProps.sys.id;
+  } catch (err) {
+    throw new Error('App definition creation failed: ' + err);
```

Comment:
```suggestion
    throw new Error(`App definition creation failed: ${err.message}`);
```

Reply:
`err.message` might need some type coercion tho cuz I think `err` is of type `unknown`